### PR TITLE
Vanilla Javascript.Fixes #264

### LINKED ISSUE #264: Add vanilla javascript
I want to add vanilla javascript. @ddvader44 please assign me this issue.

### PR DESCRIPTION
**Pull request by Khushi-agarwal**
## Explanation
Fixes #264 
Added Vanilla javascript.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR does not break any part of the existing code.
- [x] The PR creator have carefully read & adher to the contributing guidelines of the project.
- [x] The PR is made to the branch that's **called** "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] Have you followed the guidelines in the Contributing document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/CSwala/awesome-computer-science/pulls) for the same tech?
- [x] Is your submission is according to the desired pattern ?
- [x] Have you placed the contents in the `Readme.md` keeping the sorting order in mind ?
- [x] Have you placed the contents in the `Dictionary.json` keeping the sorting order in mind ?

Does you submission include -

- [x]	Category/Type
- [x]	Official Documnetation  
- [x]	Best YouTube creators  
- [x]	Useful Websites  
- [x]	Dedicated Github page 
- [x]	Best Courses  
- [x]	Ohter Tips / Hacks

### CODE REVIEW

**Comment by Khushi-agarwal** [comment]
@ddvader44 @riturajjain2000 please review this.

**Review by ddvader44 — APPROVED** [review]

File: Dictionary.json [hunk 1]
```diff
@@ -3264,6 +3264,31 @@
 		"Tips4": "https://gpuopen.com/wp-content/uploads/2016/05/Most-common-mistakes-in-Vulkan-apps.pdf",
 		"Image1": "https://en.wikipedia.org/wiki/File:Vulkan_RGB_Dec16.svg"
 	},
+	{
+		"Title": "Vanilla Javascript",
```
Comment (riturajjain2000) [inline Dictionary.json]: Change title to Vanilla JS

**Review by riturajjain2000 — CHANGES_REQUESTED** [review]
It will come before Vulkan in the list

**Comment by Khushi-agarwal** [comment]
@riturajjain2000 I have made the changes.Kindly have a look.

**Review by riturajjain2000 — APPROVED** [review]

**Pull request merged**